### DISCUSSION TITLE
Should I buy a domain name for this project?

### DISCUSSION THREAD

**Discussion by eduardconstantin** [eduardconstantin/azure-fundamentals#37] [Q&A] [answered]
Currently the project uses the free domain name provided by Azure.
I am thinking of buying a domain name for this project to make it look more professional. However, I am not sure if it is worth the cost and effort.
What should I do? Stick with the default one or buy a custom domain?

**Comment by WISNIOM** [comment, marked as answer]
I think you could buy a domain once you are sure the project is ready. For now I'd stick to the custom domain.

**Reply by eduardconstantin** [reply]
yeah, I was thinking the same. Maybe after I finish with the exam mode

**Reply by WISNIOM** [reply]
If you need help with that. or any other case you can create issue for that. I'll be free for a week or two, so I can spend some time for this project.

**Reply by eduardconstantin** [reply]
great! I'll add some issues for that mode too, I am thinking maybe I could also integrate OpenAI API to generate more questions, probably I'll create another discussion for that

**Comment by danieldanielecki** [comment]
Glad to see you're having fun creating projects from my & our community materials.
The project looks great; we might look closer into this while developing our platform for solving exams. 
I feel, that's a win-win one!

**Reply by eduardconstantin** [reply]
Yes, it is, the materials were really helpful. 
I need to find some time to start implementing the exam mode, I was thinking maybe it could be infused with generative AI.
